src/index.css

```diff
@@ -290,4 +290,16 @@ input[type='radio'] {
 /* Handle */
 ::-webkit-scrollbar-thumb {
     background: var(--grey-1-color);
+}
+
```

Comment:
Nice customized scrolls. Its good that you are even taking care of small details